### PR TITLE
Admin dashboard filter

### PR DESCRIPTION
**Pull request by SwapnaMohanty4554**
###Description

Added filter for admin.

###Screenshots

image

### CODE REVIEW

**Review by saad95544 — APPROVED** [review]
please review comments in next PR

File: src/app/components/admin/admin.component.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
           <button type="submit"><i class="fa fa-search"></i></button>
         </div>
         </form>
-        <i class="fa fa-filter admin-filter"></i>
+        <button class="fa fa-filter admin-filter" style="font-size:25px;color:rgb(133, 55, 197)" (click)="openDialog()"></button>
```
[the remark this answers is not in the document]
Comment (SwapnaMohanty4554) [reply src/app/components/admin/admin.component.html]: added

File: src/app/components/admin/admin.component.scss [hunk 2]
```diff
@@ -40,6 +40,10 @@
     border-left: 1px solid grey;
 }
 
+.admin-filter {
+    font-size: 20px !important;
```
[the remark this answers is not in the document]
Comment (SwapnaMohanty4554) [reply src/app/components/admin/admin.component.scss]: used rem

**Comment by SwapnaMohanty4554** [comment]
@saad95544 , as resolving merge conflict, have addressed the comments.

**Pull request merged**